### PR TITLE
Get Started button updates 

### PR DESCRIPTION
**Pull request by SafeEHA**
- Changed the links on the get started button and `try-it` nav link
- Deleted try-it.md and try-it.html files

### CODE REVIEW

File: try-it.md [hunk 1]
```diff
@@ -33,11 +33,24 @@ permalink: /try-it.html
...
+```
+  $ sudo visudo
+```
+- Include this line at the end of the sudoers file 
+```console
+  username  ALL=(ALL) NOPASSWD: ALL
+```
+- Save and exit
+- Manually **enable nested virtualization** if you don't have it enabled in your VM
+
```
Comment (hroyrh) [inline try-it.md:53]: Let's also include the command and output to check if its enabled or not.

Comment (SafeEHA) [reply try-it.md:53]: That would be great, but the resources i found don't point to a particular command that checks if it is enabled or not?
Can you please help with this

**Comment by SafeEHA** [comment]
Still trying to rebase

**Comment by SafeEHA** [comment]
Reopening this

**Pull request closed without merging**